shell/pages/c/_cluster/explorer/index.vue

```diff
@@ -178,16 +175,6 @@ export default {
       return this.$store.getters['management/all'](MANAGEMENT.CLUSTER);
     },
 
-    hidePspDeprecationBanner: mapPref(PSP_DEPRECATION_BANNER),
```

Comment:
Is this to be deleted in 2.8? It's still used in the code (template part)

Reply:
Alright, it looks like https://github.com/aalves08/dashboard/commit/92d810399ac7f56aad5fec7cba23ed0e2be1ab28#diff-36dedf9fa870d90a88aecacdcdc91c2c1f0043ce96183e1677c480839d475a4cL30 is what got included. 

I'll go through this pr compared to your other one line by line to make sure there wasn't any other issue. Sorry about that.